lxd/apparmor.go

```diff
@@ -105,8 +103,8 @@ func runApparmor(command string, c container) error {
 
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		shared.Log.Error("Running apparmor",
-			log.Ctx{"action": command, "output": string(output), "err": err})
+		shared.Log("error", "Running apparmor",
```

Comment:
Can we use constants (or better yet, just keep the old Error function API to reduce the diff noise) here? I don't want to miss a log statement somewhre becuase I typed "errro" instead of "error".